src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const add = (...number) => {
```

Comment:
And it'd be even better if you just pass an argument here instead of using the rest operator. You could check if it's undefined later.